.travis.yml

```diff
@@ -0,0 +1,39 @@
+# Config file for automatic testing at travis-ci.org
+
+language: python
+python: 3.6
+
+dist: trusty 
+
+matrix:
+  include:
+#    - env: TOX_ENV=py27-django19-es60 ES_APT_URL=https://artifacts.elastic.co/packages/6.x/apt
+#      python: 2.7
+    - env: TOX_ENV=py27-django19-es61 ES_APT_URL=https://artifacts.elastic.co/packages/6.x/apt
+      python: 2.7
+# TBD Python 3 tests ...
```

Comment:
If you're interested, I think you can keep these under the `allow_failures` key to still let them run without affecting the entire build's success/failure outcome.
https://docs.travis-ci.com/user/customizing-the-build/#rows-that-are-allowed-to-fail

Reply:
didn't know about that option, thanks!

Reply:
Since I'm using the `include` directive explicitly, I can't get it to work with `allow_failures` yet. I tried it in 807e30b and, and commented out the section in 963bd21 with the plan to revisit this in issue #5 and try to get the matrix to work like it's supposed to. I'll make note of this there. Thanks for the suggestion!